From: Alexis Murzeau <amubtdx@gmail.com>
Date: Sat, 4 Dec 2021 16:39:45 +0100
Subject: bash completion: remove shebang

Auto completion scripts are not meant to be executable, but only sourced.
See also: https://salsa.debian.org/lintian/lintian/-/merge_requests/292#note_139419

Forwarded: https://github.com/iterative/shtab/pull/61
Author: Alexis Murzeau <amubtdx@gmail.com>
---
 completions/bash/streamlink | 1 -
 1 file changed, 1 deletion(-)

diff --git a/completions/bash/streamlink b/completions/bash/streamlink
index 488016e..80462d0 100644
--- a/completions/bash/streamlink
+++ b/completions/bash/streamlink
@@ -1,4 +1,3 @@
-#!/usr/bin/env bash
 # AUTOMATCALLY GENERATED by `shtab`
 
 
